en/orm/table-objects.rst

```diff
@@ -325,6 +324,22 @@ atomic deletes where database operations are implicitly committed. The event is
 triggered only for the primary table on which ``delete()`` is directly called.
 The event is not triggered if a transaction is started before calling delete.
 
+Stopping Table Events
+---------------------
+To prevent the saving from continuing simply stop event propagation in your callback::
```

Comment:
Suggestion:

To prevent the save from continuing,

Reply:
You can directly add commits on top if you want. Your English is supposed to be always better than mine anyway :P

Reply:
I don't think I can yet.  Your english is great! I just try to make things easier to read :)